### PR TITLE
Rework configuration UI

### PR DESCRIPTION
**Pull request by jbaiter**
- Configuration is now performed via an on-screen bubble, similar to mirador-image-tools. This should make it easier to toggle settings on the fly, useful e.g. on mobile devices were disabling selection might be necessary to be able to pan and zoom
- The plugin menu in the window top bar is completely gone
- The `windowTextSettings` store state and its corresponding actions, selectors and sagas have been removed in favor of a `textOverlay` key in the window's configuration (which is how it's configured anyway).

Here's a quick video of the new UI in action:

new_ui

### CODE REVIEW

File: src/components/TextOverlaySettingsBubble.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import { Slider } from '@material-ui/core';
+import TextIcon from '@material-ui/icons/TextFields';
+import CloseIcon from '@material-ui/icons/Close';
+import SubjectIcon from '@material-ui/icons/Subject';
+import OpacityIcon from '@material-ui/icons/Opacity';
+import TextSelectIcon from './TextSelectIcon';
+
+/** Control text overlay settings  */
+const TextOverlaySettingsBubble = ({
+  windowTextOverlayOptions, imageToolsEnabled, textsAvailable, updateWindowTextOverlayOptions, t,
```
Comment (morpheus-87) [inline src/components/TextOverlaySettingsBubble.js:13]: Wo kommt denn das `t` her?

File: src/components/TextOverlaySettingsBubble.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      && (
+      <>
+        <div style={{
+          borderImageSlice: 1,
+          borderImageSource: 'linear-gradient(to bottom, rgba(0, 0, 0, 0) 20%, rgba(0, 0, 0, 0.2) 20% 80%, rgba(0, 0, 0, 0) 80% )',
+          borderRight: '1px solid rgba(0, 0, 0, 0.2)',
+          display: 'inline-block',
+          paddingRight: 8,
+        }}
+        >
```
Comment (morpheus-87) [inline src/components/TextOverlaySettingsBubble.js:44]: Nit: vielleicht in einer Zeile?

File: src/components/TextOverlaySettingsBubble.js (hunk 1 above)
Comment (jbaiter) [reply src/components/TextOverlaySettingsBubble.js:13]: Das wird von Mirador immer an Plugins übergeben um Zugriff auf die i18n-Infrastruktur zu geben.

File: src/components/TextOverlaySettingsBubble.js (hunk 2 above)
Comment (jbaiter) [reply src/components/TextOverlaySettingsBubble.js:44]: Hm, das hat eslint so gewollt (`react/jsx-closing-bracket-location` Regel), sollen wir da die Presets verändern?

File: src/components/TextOverlaySettingsBubble.js (hunk 1 above)
Comment (morpheus-87) [reply src/components/TextOverlaySettingsBubble.js:13]: Gut zu wissen :)

File: src/components/TextOverlaySettingsBubble.js (hunk 2 above)
Comment (morpheus-87) [reply src/components/TextOverlaySettingsBubble.js:44]: Kannst du machen, wenn du magst, muss aber nicht unbedingt.

**Pull request merged**